app/src/main/AndroidManifest.xml

```diff
@@ -23,10 +23,12 @@
         android:logo="@mipmap/ic_launcher"
         android:requestLegacyExternalStorage="true"
         android:theme="@style/OpeningTheme"
+        android:resizeableActivity="true"
         tools:ignore="AllowBackup">
         <activity
             android:name=".MainActivity"
             android:label="@string/app_name"
+            android:configChanges="orientation|screenSize|smallestScreenSize|density|screenLayout|uiMode|keyboard|keyboardHidden|navigation"
```

Comment:
Not sure about this: https://developer.android.com/guide/topics/manifest/activity-element#config